geps/gep-1731/index.md

```diff
@@ -0,0 +1,415 @@
+
+## Non-Goals
+
+* To allow more granular control of the backoff strategy than many dataplanes allow customizing, such as whether to use an exponential backoff interval between retry attempts, add jitter, or cap the backoff interval to a maximum duration.
+* To allow specification of a default retry policy for all routes in a given namespace or attached to a particular Gateway.
+* A standard API for approaches for retry logic other than max count or "budget", such as interaction with rate limiting headers.
+
+## Introduction
+
+TODO
+
+### Background on implementations
+
+Most implementations that handle HTTPRoute objects use a proxy as the data plane
+implementation, which forwards traffic flows as directed by Gateway API configuration.
+
+The following table is a review of all the listed implementations of Gateway API
+at the time of writing, with the data plane they use for Layer 7, based on what information
+could be found online. If there are errors here, or if the implementation doesn't
+support Layer 7, please feel free to correct them.
+
+| Implementation                           | Data Plane       |
+|------------------------------------------|------------------|
+| Acnodal EPIC                             | Envoy            |
+| Apache APISIX                            | NGINX            |
+| Azure Application Gateway for Containers | Envoy            |
+| BIG-IP Kubernetes Gateway                | F5 BIG-IP        |
+| Cilium                                   | Envoy            |
+| Contour                                  | Envoy            |
+| Emissary Ingress                         | Envoy            |
+| Envoy Gateway                            | Envoy            |
+| Flomesh Service Mesh                     | Pipy             |
+| Gloo Edge                                | Envoy            |
+| Google Kubernetes Engine (GKE)           | Similar to Envoy |
+| HAProxy Ingress                          | HAProxy          |
+| Hashicorp Consul                         | Envoy            |
+| Istio                                    | Envoy            |
+| Kong                                     | NGINX            |
+| Kuma                                     | Envoy            |
+| Linkerd                                  | linkerd2-proxy   |
+| Litespeed                                | Litespeed WebADC |
+| NGINX Gateway Fabric                     | NGINX            |
+| Traefik                                  | Traefik          |
+| VMWare Avi Load Balancer                 | ?                |
+
+Implementation retry configuration details below have been summarized or copied verbatim from corresponding project documentation, with links to the original source where applicable.
+
+<details>
+  <summary>Envoy</summary>
```

Comment:
Not sure if this is a website bug or user error but the markdown within this doesn't render on the site FYI

Reply:
Maybe doesn't handle mixed HTML

Reply:
Ahh alright - it's dependent on the Markdown parser so GitHub flavored Markdown does support it, will clean this up so it's clear on web.